tensorflow/python/ops/histogram_ops.py

```diff
@@ -88,6 +88,9 @@ def histogram_fixed_width_bins(values,
     # map tensor values within the open interval value_range to {0,.., nbins-1},
     # values outside the open interval will be zero or less, or nbins or more.
     indices = math_ops.floor(nbins_float * scaled_values, name='indices')
+    
+    if value_range[0].numpy() == value_range[1].numpy():
```

Comment:
tensor.numpy() is only available when eager execution is enabled (not while building graph functions)

So please write this using array_ops.where or control_flow_ops.cond